src/talos/RunHailFiltering.py

```diff
@@ -517,8 +507,11 @@ def filter_by_consequence(mt: hl.MatrixTable) -> hl.MatrixTable:
         ),
     )
 
-    # filter out rows with no tx consequences left, and no splice cat. assignment
-    return filtered_mt.filter_rows(hl.len(filtered_mt.transcript_consequences) == 0, keep=False)
+    # filter out rows with no high impact tx consequences, and no clinvar Pathogenic annotation
+    return filtered_mt.filter_rows(
+        (hl.len(filtered_mt.transcript_consequences) == 0) & (mt.clinvar_talos == 0),
```

Comment:
I agree we should retain clinvar variants, but why did we stop retaining sAI variants? We really want to be able to use this cat as an option.

Reply:
We stopped including SpliceAi because we were previously getting all those annotations from the `seqr_loading_pipelines` annotation resource, which wasn't realistic to distribute. The bones of the category still exist in the codebase, so if we have a more portable source for that annotation we can add it back in. That'll mean anyone able to use it without violating license terms can include it, but it'll be optional for everyone else